static/js/dom.js

```diff
@@ -19,39 +19,170 @@ export let dom = {
+        $(document).ready(function () {
+            $('.board-toggle').click(function () {
+                $('#box' + $(this).attr('target')).slideToggle(400);
+            });
+        });
     },
     // here comes more features
+
+    login: function () {
+
+        let data = {
+            "title": "Login",
+            "button_text": "Login",
+            "url": "/login",
+            "message_success": "Logged in ",
+            "message_fail": "Wrong user name or password"
+        };
+
+        dom.getAjax(data);
+    },
+
+
+    register: function () {
+
+        let data = {
+            "title": "Register new user",
+            "button_text": "Register",
+            "url": "/register",
+            "message_success": "Register succesfull",
+            "message_fail": "Register failed"
+        };
+
+        dom.getAjax(data);
+    },
+
+    getAjax: function (data) {
+        event.preventDefault();
+        let form_values = {};
+
+
+        $('#inputLabel').text(data["title"]);
+        $('#submit-button').text(data["button_text"]);
+
+
+        $('#inputModal').modal({show: true});
+
+        $('#submit-button').click(function () {
+            let $inputs = $('#inputForm :input');
+            $inputs.each(function () {
+                form_values[this.name] = $(this).val();
+            });
+
+            $.ajax({
+                type: 'POST',
+                url: data['url'],
+                dataType: 'json',
+                data: form_values
+            })
+                .then(
+                    function success(data) {
+
+                        sessionStorage.setItem('username', form_values["user-name"]);
+                        sessionStorage.setItem('userid', data["userid"]);
+                        dom.showLoggedIn(form_values["user-name"], true);
+                        location.reload()
+                    },
+                    function fail() {
+                        alert(data["message_fail"]);
+                        location.reload();
+                    }
+                );
+
+
+        });
+    },
+    showLoggedIn: function () {
+        let username = sessionStorage.getItem("username");
+        let register = document.getElementById("register");
+        let login = document.getElementById("login");
+        let logout = document.getElementById("logout");
+        let navbar = document.getElementById("navbar-text");
+        if (username) {
+            navbar.style.display = 'block';
+            navbar.innerText = `Signed in as ${username}`;
+            register.style.display = 'none';
+            login.style.display = 'none';
+            logout.style.display = 'block';
+        } else {
+            navbar.style.display = 'none';
+            register.style.display = 'block';
+            login.style.display = 'block';
+            logout.style.display = 'none';
+        }
+    },
+
+    logout: function () {
+        if (sessionStorage.getItem("username"))
```

Comment:
You don't need to check that `getItem` gives back a truthy value, you can go ahead and `removeItem`